packages/paste-website/src/utils/DataUtils.ts

```diff
@@ -0,0 +1,85 @@
+import camelCase from 'lodash/camelCase';
+import mapKeys from 'lodash/mapKeys';
+
+import {getHumanizedNameFromPackageName, getNameFromPackageName} from './RouteUtils';
+import {sentenceCase} from './SentenceCase';
+
+export type ApiData = Record<string, any>;
+
+type MutationFunction = (data: ApiData) => ApiData;
+
+const KEYS_TO_FLATTEN = new Set([
+  'allAirtable',
+  'allPasteComponent',
+  'allPasteDesignTokensPackage',
+  'allPasteLayout',
+  'allPasteLibraries',
+  'allPastePattern',
+  'allPastePrimitive',
+  'allPasteThemePackage',
+]);
+
+const shouldFlatten = (key: string): boolean => KEYS_TO_FLATTEN.has(key);
+
+export const getNormalizedHeaderData = (data: ApiData): ApiData => {
```

Comment:
quibble: can we add comments in this file for the exported functions for what their purpose is, and where they're used please?